stormforge/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "6f1ddcc9-e704-4f94-941b-8a914fcd89a0",
+    "app_id": "stormforge",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Real-time optimization of Kubernetes resources using machine learning",
+        "title": "StormForge",
+        "media": [{
+                "media_type": "image",
+                "caption": "Create an Application as a collection of resources to target for optimization.",
+                "image_url": "images/sf_ui_01_application.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Configure Optimize Live",
+                "image_url": "images/sf_ui_02_configure.jpg"
+            }
+        ],
+        "classifier_tags": [
+            "Supported OS::Linux",
+            "Category::Cloud",
+            "Category::Configuration Deployment",
```

Comment:
Value of parameter 'classifier_tags' should be any of these ['Category::Alerting', 'Category::Autodiscovery', 'Category::Automation', 'Category::AWS', 'Category::Azure', 'Category::Caching', 'Category::Cloud', 'Category::Collaboration', 'Category::Compliance', '**Category::Configuration & Deployment**', 'Category::Containers', 'Category::Cost Management', 'Category::Data Store', 'Category::Developer Tools', 'Category::Event Management', 'Category::Exceptions', 'Category::Google Cloud', 'Category::Incidents', 'Category::IOT', 'Category::ISP', 'Category::Issue Tracking', 'Category::Languages', 'Category::Log Collection', 'Category::Mainframe', 'Category::Marketplace', 'Category::Messaging', 'Category::Metrics', 'Category::Mobile', 'Category::Monitoring', 'Category::Network', 'Category::Notification', 'Category::Oracle', 'Category::Orchestration', 'Category::OS & System', 'Category::Processing', 'Category::Profiling', 'Category::Provisioning', 'Category::SAP', 'Category::Security', 'Category::SNMP', 'Category::Source Control', 'Category::Testing', 'Category::Tracing', 'Category::Web', 'Offering::Integration', 'Offering::Professional Service', 'Offering::Software License', 'Offering::UI Extension', 'Supported OS::Android', 'Supported OS::Any', 'Supported OS::HP-UX', 'Supported OS::IBM i/OS', 'Supported OS::IBM z/OS', 'Supported OS::iOS', 'Supported OS::Linux', 'Supported OS::macOS', 'Supported OS::Solaris', 'Supported OS::Windows', 'Supported OS::Mac OS', 'Category::OS System']

Reply:
Updated to "Configuration & Deployment"